src/components/Products/Products.jsx

```diff
@@ -0,0 +1,15 @@
+import CommonBanner from '@/UI/CommonBanner';
+import React from 'react';
+
+const Products = () => {
+    return (
+        <div>
+            <CommonBanner
+            title='Doctors Prescribe Products'
+            routeName='Products'
+            />
+        </div>
+    );
+};
```

Comment:
`
const Products = () => 
    (
            <CommonBanner
            title='Doctors Prescribe Products'
            routeName='Products'
            />
    );

`
import like above if don't need the div

Reply:
We need it, because we will have more divs after commonBanner for the other sections